lib/Current.dart

```diff
@@ -0,0 +1,706 @@
+                child: Text(""),
+              ),
+            ],
+          ),
+        ),
+        Expanded(
+          flex: 9,
+          child: Column(
+            // mainAxisAlignment: MainAxisAlignment.center,
+            mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              Text('$title\n $subTile',
+                  style: TextStyle(
+                      fontFamily: "regular",
+                      fontSize: 19,
+                      color: Colors.black54)),
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget timelineCurrentRow(String title, String subTile) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.start,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        SizedBox(height: 30),
+        Expanded(
+          flex: 1,
+          child: Column(
+            // mainAxisAlignment: MainAxisAlignment.start,
+            mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: <Widget>[
+              ClipRRect(
+                borderRadius: BorderRadius.circular(5.0),
+                child: Container(
+                  width: 18,
+                  height: 18,
+                  decoration: new BoxDecoration(
+                    color: Colors.black,
+                    shape: BoxShape.circle,
+                    border: new Border.all(
+                        color: Colors.white,
+                        width: 6.0,
+                        style: BorderStyle.solid),
+                    boxShadow: [
+                      new BoxShadow(
+                        color: Colors.grey[300],
+                        blurRadius: 30.0,
+                      )
+                    ],
+                  ),
+                  child: Text(
+                    "",
+                  ),
+                ),
+              ),
+              Container(
+                width: 3,
+                height: 160,
+                decoration: new BoxDecoration(
+                  color: Colors.black,
+                  shape: BoxShape.rectangle,
+                ),
+                child: Text(""),
+              ),
+            ],
+          ),
+        ),
+        Expanded(
+          flex: 9,
+          child: Column(
+            // mainAxisAlignment: MainAxisAlignment.center,
+            mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              Text('$title',
+                  style: TextStyle(
+                      fontFamily: "regular",
+                      fontSize: 19,
+                      color: Colors.black)),
+              Column(
+                children: <Widget>[RideStatus()],
+              )
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget timelineLastRow(String title, String subTile) {
```

Comment:
Idk if it is caused by this widget, but there is a 21 pixel overflow underneath it when I run the app. Try fixing this by looking for places that can be more responsive. Also consider the case where the data on the screen takes up much more space than when it's hardcoded. Unless I missed where you did it (and tell me if I did), you should handle overflowing content perhaps by making that container scrollable? You can also ask the designers for advice.